pyiceberg/table/__init__.py

```diff
@@ -191,6 +193,15 @@ class TableProperties:
     DELETE_MODE_MERGE_ON_READ = "merge-on-read"
     DELETE_MODE_DEFAULT = DELETE_MODE_COPY_ON_WRITE
 
+    READ_SPLIT_SIZE = "read.split.target-size"
```

Comment:
lets add these options to the docs similar to https://py.iceberg.apache.org/configuration/#write-options

Reply:
looks like these are modelled after spark configs https://github.com/apache/iceberg/blob/main/docs/docs/spark-configuration.md#read-options

Reply:
added to docs, those options exist in table properties as well https://iceberg.apache.org/docs/latest/configuration/#read-properties